redshift_sqlalchemy/dialect.py

```diff
@@ -189,17 +279,106 @@ def set_isolation_level(self, connection, level):
 
         connection.set_isolation_level(level)
 
-    def _get_column_info(self, *args, **kwargs):
+    def _get_column_info(self, col, domains):
```

Comment:
I intensely dislike *args and **kwargs. They can't always be avoided, but if they can I think it improves code readability and debugability.